docs/content/Auth/Overview.md

```diff
@@ -42,6 +42,19 @@ Authentication is handled outside of Cube.js. A typical use case would be:
 
 ## Using JSON Web Key Sets (JWKS)
 
+### Guides
+
+Cube.js has out-of-the-box support for the following identity providers:
+
+- [Auth0][ref-jwt-auth0]
+- [AWS Cognito][ref-jwt-aws-cognito]
+
+If you don't see your identity provider listed, please let us know on Discourse
```

Comment:
🙌 Do we want to suggest a specific category in Discourse? Maybe `Use Cases` or `Ideas`?

Reply:
I was thinking Ideas, but I'm totally open to suggestions.

Thoughts @keydunov, @igorlukanin?